src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -1,9 +1,11 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import getProp from 'lodash/get';
+import fuzzysearch from 'fuzzysearch';
```

Comment:
Can we use our own fuzzy search [utility](https://github.com/box/box-ui-elements/blob/master/src/utils/fuzzySearch.js) rather than adding a third-party dependency?